services/nginx/nginx.conf

```diff
@@ -0,0 +1,21 @@
+server {
```

Comment:
is there a way to test this with cypress? just a simple healthcheck type thing for now

Reply:
yeah good point, i'll update the url for cypress health check tests. If it can resolve to that endpoint of `localhost/` and `localhost/api/health-check` that signifies nginx is forwarding the request properly. 
As for db, we have no data in it right now since we don't even have a table, once we have some data we can seed some data and query an endpoint to check if the api returns the seed data.

Reply:
ahh the only reason I didn't update the cypress tests was because we haven't figured out how to spin up containers in github actions.

Reply:
the e2e testing on github actions is containerized as well now, so all testing urls are pointing to the ones being handled by the nginx container.